.github/workflows/ci-pr.yaml

```diff
@@ -28,7 +28,8 @@ jobs:
         repository: ${{github.event.pull_request.head.repo.full_name}}
     - uses: actions/setup-dotnet@v2
       with:
-        dotnet-version: '6.0'
+        dotnet-version: '7.0'
+        include-prerelease: true
```

Comment:
Question: @mathieu-benoit, do you recommend removing `include-prerelease: true` when we migrate to the a non-preview version of .NET 7 (in Nov)?

Reply:
Yep, I will remove them as soon as .NET 7 is GA ;)